### PR TITLE
STORM-1009: link on http://storm.apache.org/documentation/Kestrel-and…

### PR DESCRIPTION
**Pull request by jayapriya90**
…-Storm.html is wrong

### CODE REVIEW

File: documentation/Kestrel-and-Storm.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This page explains how to use to Storm to consume items from a Kestrel cluster.
 
 ## Preliminaries
 ### Storm
-This tutorial uses examples from the [storm-kestrel](https://github.com/nathanmarz/storm-kestrel) project and the [storm-starter](http://github.com/apache/storm/blob/master/examples/storm-starter) project. It's recommended that you clone those projects and follow along with the examples. Read [Setting up development environment](https://github.com/apache/storm/wiki/Setting-up-development-environment) and [Creating a new Storm project](https://github.com/apache/storm/wiki/Creating-a-new-Storm-project) to get your machine set up.
+This tutorial uses examples from the [storm-kestrel](https://github.com/nathanmarz/storm-kestrel) project and the [storm-starter](http://github.com/apache/storm/blob/master/examples/storm-starter) project. It's recommended that you clone those projects and follow along with the examples. Read [Setting up development environment](Setting-up-development-environment.html) and [Creating a new Storm project](https://github.com/apache/storm/wiki/Creating-a-new-Storm-project) to get your machine set up.
```
Comment (wuchong) [inline documentation/Kestrel-and-Storm.md]: the document <Creating a new Storm project> link maybe need to be changed as well `[Creating a new Storm project](Creating-a-new-Storm-project.html)`

**Comment by jayapriya90** [comment]
@wuchong Thanks for the review. I've updated pull request with the necessary changes. Can you please have a look at it?

**Comment by kishorvpatil** [comment]
LGTM. +1

**Comment by wuchong** [comment]
LGTM. +1

**Comment by HeartSaVioR** [comment]
+1

**Comment by jerrypeng** [comment]
+1

**Comment by haohui** [comment]
+1 can you please squash the two commits into one?

**Comment by revans2** [comment]
+1

**Pull request merged**